File: Flee/ListProperty.vb
Public Class ListProperty

	Public name As String
	Public value As String

	Public Sub New(line As String)
		If line(0) = "\t" OrElse line(0) = " " Then
			Throw New Exception("Malformed property: " + line)
		End If
		Dim tuple() As String = line.Split("=")
		Me.name = tuple(0)
		Me.value = tuple(1)
	End Sub

End Class

File: Flee/ListProperty.cs
using System;
using Microsoft.VisualBasic.CompilerServices;

namespace Flee {
	public class ListProperty {
		public string name;
		public string value;

		public ListProperty(string line) {
			if (line[0] == '\t' || line[0] == ' ')
				throw new Exception("Malformed property: " + line);

			var tuple = line.Split(new char[] { '=' }, 2);
			name = tuple[0];
			value = tuple[1];
		}
	}
}